watertap/unit_models/ion_exchange_0D.py

```diff
@@ -544,36 +609,22 @@ def build(self):
             initialize=0.0086,
             bounds=(0, 0.01),  # MWH, Perry's
             units=pyunits.m / pyunits.s,
-            doc="Velocity through resin bed",
+            doc="Superficial velocity through bed",
         )
 
         self.vel_inter = Var(
             initialize=0.01,
             units=pyunits.m / pyunits.s,
-            doc="Interstitial velocity",
-        )
-
-        self.holdup = Var(
-            initialize=100,
-            bounds=(90, 250),  # Inamuddin/Luqman
-            units=pyunits.dimensionless,
-            doc="Holdup percent",
+            doc="Interstitial velocityv through bed",
         )
 
         self.service_flow_rate = Var(
             initialize=10,
-            bounds=(1, 15),
+            bounds=(1, 40),
             units=pyunits.hr**-1,
             doc="Service flow rate [BV/hr]",
         )
 
-        self.pressure_drop = Var(
-            initialize=14,
-            units=pyunits.psi,
-            bounds=(0, 25),
-            doc="Pressure drop across column",  # max pressure drop is 25 psi, MWH
-        )
-
         # ====== Dimensionless variables ====== #
 
         self.Re = Var(
```

Comment:
```suggestion
        self.N_Re = Var(
```

Reply:
modified all dimensionless numbers to match this format